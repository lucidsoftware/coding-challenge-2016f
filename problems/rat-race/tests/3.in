2
51 51
###################################################
#.....#.#...#.............................#.......#
#.###.#.#.###.#####.#.#####.###############.#.###.#
#...#.#.#...#.#...#.#.....#...#.....#.#.....#...#.#
###.#.#.#.###.#.#.#######.###.#.#.#.#.#.#######.#.#
#...#...#...#.#.#.#.......#...#.#.#...#.#...#...#.#
#.#########.#.#.#.#.###########.#.#####.#.#.#.#.###
#.#.........#.#.#.#.....#...#...#.#.....#.#...#...#
#.#.#####.###.###.#####.#.#.#.###.#.#####.#####.#.#
#.....#.....#...#.....#...#...#...#.#.....#.....#.#
#.###.#.###.###.#####.#####.###.###.#.###########.#
#.#.#.#.#...#...#.........#.#...#...#...#.......#.#
###.#.#.#####.#####.###.#####.#.#.###.###.#####.###
#...#.#.....#.#.....#.#.#...#.#.#...#.#...#...#...#
#.###.###.#.#.#.#####.#.#.#.#.###.###.#.###.#.###.#
#.#...#.#.#.#...#.#...#.#.#.#.....#...#.....#.#...#
#.#.###.#.#######.#.#.#.#.#.#####.#.###.#######.###
#.#.....#...#...#...#.#.#.#.....#.#.......#...#...#
#.#.#.###.###.#.#.###.#.#.#####.#########.#.#.###.#
#...#.#...#...#...#.#...#.#.............#.#.#.....#
###.###.#.#.#######.#####.#.###########.###.#####.#
#...#...#.#.............#.#...#...#...#.#...#...#.#
#####.###.#####.#######.#.###.#.#.#.#.#.#.###.#.#.#
#.....#.....#...#.......#...#.#.#...#.#.#...#.#...#
#.#.#########.#############.###.#####.#.###.#.#####
#.#...#.......#...........#.....#...#.#.....#...#.#
#.#####.#.#####.#########.#######.#.#.#########.#.#
#.....#.#...#...#.......#.#.......#.#.....#...#...#
#####.#.#####.#####.#######.#######.#.###.#.#.###.#
#.....#.....#...#.......###...#...#.#...#...#.#...#
#.#####.#.#####.#.#####.###.#.#.#.#.#########.#.###
#.......#...#...#...#.#.#...#.#.#.#.....#.....#.#.#
#############.#####.#.#.#.###.###.#####.#.#####.#.#
#.............#.#...#.#...#...#.....#...#...#.....#
#.#############.###.#.###.#.###.###.#.#####.#.#####
#.........#.........#...#.#.#...#...#...#...#.#...#
#########.#######.#####.###.#.#.###.###.#.###.#.#.#
#.......#.....#.........#...#.#...#.#...#.#...#.#.#
#.###.#######.###########.###.###.###.###.###.#.###
#.#...#.....#.........#.#...#.#...#.#...#...#.#...#
###.#.#.#############.#.###.###.###.###.#.#.#.###.#
#...#.#.......#.....#.#.....#...#.......#.#.#.....#
#.###.#####.#.#.#####.#.#####.###.#########.#####.#
#.#.......#.#.#.......#...#...#...#...#...#.#.....#
#.#####.#.#.#.#.#######.#.###.#.###.#.###.#.###.###
#.....#.#...#.#.......#.#.#...#...#.#.......#...#.#
#####.###############.###.#.#####.#.#########.###.#
#####.............#.#.....#.#.....#.#.......#...#.#
###########.#.###.#.#######.#.#####.#.###.#####.#.#
#########...#...#...................#...#.........#
###################################################
101 101
...#...#.........#...........#.#...#...#.......#.........#.......#.#.......#...#.....#...........#...
.#.#.#.#.#######.#.#########.#.#.#.#.#.###.#.#.#######.#.#.#####.#.#.###.#.###.#.#.#####.#######.#.#.
.#.#.#.......#...#.#.......#...#.#...#.....#.#.........#.#.#...#.#.....#.#.......#.#...#.....#.#.#.#.
.#.#######.#.#.###.#####.#####.#.###.#####.#############.#.#.#.#.#######.#########.#.#.#####.#.#.#.#.
.#.........#.#...#...#.....#...#...#.#.....#.....#...#...#.#.#.#...........#...#...#.#...#...#.#.#.#.
.#############.#####.#####.#.#####.#.#.#####.###.#.#.#####.#.###.#####.#####.#.#####.###.#.###.#.#.##
.............#.#.....#...#.#...#...#.#...#...#...#.#.......#.#...#.#.....#...#.#.....#.....#...#.#...
######.#.###.#.#.#####.#.#.###.#.###.#####.###.###.#########.#.###.#.#####.#.#.#.###.#######.#.#.###.
.....#.#.#...#.#...#...#.#...#...#.#.........#...#.#.......#.#...#...#...#.#.#.#.#.#.#...#...#.#.....
.###.#####.#######.###.#.###.#####.#############.#.#.#####.#.###.#.###.#.#.###.#.#.#.#.#.#.###.#####.
.#.....#...#.......#...#.....#.....#...#.......#...#.....#.#...#.#.#...#...#...#...#.#.#...#...#.....
.#####.#.#.#.#######.#######.#####.#.#.#.###.###########.#.###.#.###.#######.###.###.#######.#####.##
...#.....#.#...#.#...#.....#.#.......#.#.#.#.#...........#.....#.......#...#...#.#.....#...#.....#...
##.###########.#.#.#######.#.#########.#.#.#.#.#.#########.#.#############.###.#.#.#####.#.###.#.###.
...#.......#...#.......#...#.#...#...#.#...#...#.........#.#...#.......#.....#...#...#...#...#.#...#.
.###.#####.#.#########.#.###.#.#.#.#.#.#########.###.###.#.###.#.#####.#.###.###.###.#.#####.#####.#.
.....#...#...#.........#.....#.#...#.......#...#.#...#...#...#...#...#.#...#...#...#.#.#...#.....#...
.#####.#.#####.#############.#.###########.#.#.#######.###########.###.###.#########.#.###.#####.#.##
.....#.#.#.......#.....#.....#.#.........#...#.#.....#.#.....#...#.....#.#.#...#.....#.......###.#...
####.#.#.#.#####.###.###.#####.###.#.#########.#.###.#.#.###.###.#.#####.#.#.#.#.#####.#########.####
.....#.#.#.....#...#.......#...#...#.........#.#.#...#.#.###...#.......#.#...#.#...#...#...#...#.....
####.#.#.#####.###.#.#######.#####.#####.#####.#.###.#.#.#####.#######.#.#.###.###.#.###.#.#.#.#.###.
...#...#.#.......#.#.#.........#.#.....#...#...#.#.#.#.....#.#...#.#...#...#...#.#...#...#...#.#...#.
.#.#####.###.#####.#.#########.#.#####.###.#.###.#.#.#####.#.###.#.#.###.###.#.#.#####.#.#####.#####.
.#.#...#...#.#.....#...........#.#...#...#.#...#.#.#.#...#...#...#.#...#.#...#.....#.#.#.#...#.......
.###.#.###.#.#.#################.#.#####.#.###.#.#.#.#.###.###.#.#.###.###.#######.#.#.###.#.#######.
.#...#...#.#.#.#.....#.....#...#.....#.#.#...#.#.#...#.#.....#.#.#.....#...#.#.....#.#.....#.#.......
.#.#.#.###.#.#.#.###.#.#####.###.#####.#.###.#.#.###.#.#.###.#.###.#####.###.#.#####.#.#####.########
...#.#.#...#.#.#...#.#.#.........#.....#.#...#...#...#...#...#...#.......#...#.......#.....#...#.#...
.#####.#.###.#.###.###.#.#.#####.#.#.###.#.#######.#######.#####.#.#######.#.#.###########.###.#.#.#.
.#...#.#.#.#.#.#...#.....#.#.....#.#.....#.....#...#.....#.#...#.#.#...#...#...#.........#...#.....#.
##.#.#.#.#.###.#.###.#####.#.###.#.#########.#.#.###.###.###.#.#.#.#.#.#########.#######.###.#######.
...#...#.......#.....#.....#...#.#.#.....#.#.#.#.....#...#...#...#.#.#.....#.....#.....#...#.#.......
.#.#############.#####.###########.#.###.#.#.#########.###.#######.#.#####.#.#####.###.###.###.###.##
.#.....###.#...#...#.#.............#...#.#.#.........#.#...#.......#...#.#...#...#...#...#.....#.#...
.#####.###.#.#.###.#.###############.###.#.#.#########.###.#.#########.#.#####.#####.#.#########.###.
.#...#.....#.#.....#.#...#.......#.#...#...#.#...#.........#.#.........#...#.....#...#...#.....#.#...
.#.#.###.###.#######.#.#.###.###.#.#.###.#.#.#.#.#.#.#########.#########.#.#.###.#.#####.#.###.#.#.##
...#.#.....#.........#.#.#...#...#.....#.#.#.#.#...#.#.#.#.....#.........#.#...#...#...#...#...#.....
####.#.###.#########.#.###.###.#.#######.#.#.#.#######.#.#.#.###.#.###########.#####.#.#####.#.#####.
.....#...#.#.....#.#.#.....#...#.......#.#.#...#.....#.....#...#.#...........#...#...#.....#.#.....#.
##.#####.###.###.#.#.#.#.#############.#.#.#####.###.#.#######.###########.#####.#.#.###.#.#.#######.
.....#...#...#.#.#...#.#.#.........#.....#.....#.#...#.#...#.#...#.....#.......#.#.#...#.#.#.#.......
######.###.#.#.#.#.###.#.#.#######.#.###########.#.###.#.#.#.###.#.###.#.#####.#.#######.#.#.#.######
...#...#...#.#...#.#...#.#...#...#.#...#...#.....#...#.#.#...#...#.#...#...#...#.#.....#.#.#...#.....
.#.#.#.#.###.#####.#####.###.###.#.###.#.#.#.#######.#.#.###.#.###.#.#####.#.###.#.###.#.#.#####.###.
.#...#.#.#...#.....#.......#.....#...#.#.#...#.#.#...#.#.#.#.#.#...#...#.#.#.....#.#...#.#...#...#...
.###.###.#.###.#####.###########.###.###.#####.#.#.#.#.#.#.#.#.#.###.###.#.#######.#.###.###.#####.##
...#.#...#...#...#...#...#.....#...#.....#.......#.#.#...#.#.#.....#.....#...#.....#.#...#...#.......
.#.###.###.#####.#.#.#.###.###.###.#######.#####.#.#######.#.#########.#####.#.#####.#.#.#.#####.###.
.#...#...#.#.....#.#.#.....#...#.#.#.......#...#.#...#...#...#...#.....#...#...#...#.#.#.#.#.....#...
.###.###.#.#.#####.#.#.#####.#.#.#.#####.###.###.#####.###.###.#.#.#####.#.#######.#.###.#.#.#.######
...#.#...#.#.#.....#.#...#...#.#.......#...#.#...#.......#.#...#.#...#...#.........#.#...#.#.#.......
.###.#.###.#.#.#####.###.#.###########.###.#.#.#####.###.#.###.#.###.#.#########.#.#.###.#.#########.
.#...#...#...#.###.#.#.#.#.........#.#...#.#.......#...#...#...#...#...........#.#.#.....#...........
##.#####.#####.###.#.#.#.#######.#.#.###.#.#######.#####.###.#################.#.#########.#########.
...#.....#...#.#...#...#.....#...#...#...#...#...#.....#.....#.....#.........#.#.#.....#...#...#...#.
##.#.#####.#.#.#.#.#.#######.#.###.###.#####.###.#####.#.#####.###.#.#######.#.#.#.###.#.###.#.#.#.#.
...#.#...#.#.#...#.#.........#.#.#...#.#.....#...#...#.#.#.....#...#.#...#...#.#...#...#.#...#.#.#.#.
.###.#.#.#.#.#####.#########.#.#.###.#.#.#####.###.#.#.###.#####.###.#.#.#.#########.#.###.###.#.#.##
.....#.#...#.......#...#...#.#.#.....#...#...#.....#...#...#.#...#...#.#.#.......#.#.#.......#...#...
.#########.###########.#.#.###.#.#########.#.#.#########.###.#.###.###.#########.#.#.#######.#######.
.......#.#.#...#.....#.#.#.....#.#.......#.#.#...#.......###.#...#.#...#...#...#...#...#...#...#...#.
######.#.#.#.###.###.#.#.#######.###.#.#.#.#.###.###.###.###.###.#.###.#.###.#.###.#.###.#.#####.#.#.
.......#...#.....#.#...#.......#...#.#.#...#...#...#.#...#.....#...#...#.....#...#.#.#...#...#...#...
.#######.###.###.#.###.#######.###.###.#######.###.###.###.###.#####.###.###.#.#.#.###.#####.#.#.####
.#.......#...#.#.#...#.#.#...#.#.....#.......#...#.....#.....#.#...#.....#.#.#.#.#.....#.#...#.#...#.
.#.#########.#.#.###.#.#.#.#.#.#####.#######.#.#.#######.#####.#.#.#.#####.#.###########.#.###.###.#.
.#.#.......#...#...#.....#.#...#...#.....#.#.#.#.#...#.#...#...#.#...#.....#...#...#...#...#.#.#...#.
.#.###.###.#######.#######.#####.#.#####.#.#.#.###.#.#.#.###.###.#####.###.###.#.#.#.###.###.###.###.
.#.#...#.........#.#.....#...#...#...#.#.#...#.....#...#.#.....#.#...#.#.....#.#.#...#...#.......#...
.#.#.###########.#.#.###.###.#.#####.#.#.###.#######.###.#.#####.###.###.#####.#.###.#.#######.###.##
.#...#...#.....#...#.#.#.#.........#.#.#.#...#.....#...#.#...............#.....#.#.#...#...#...#.....
.#####.#.###.#.#####.#.#.###########.#.#.###.#.#.###.#.###############.#.#.#####.#.#####.#.###.#.###.
...#...#.....#.....#.#.#.............#.#.#.#...#...#.#...........#.#...#.#.....#.#.......#.#...#.#...
##.#.#############.#.#.###############.#.#.#######.#.#.#.#######.#.###########.#.###.#####.#.###.#.##
.#.#.....#.......#...#...............#...#.......#.#.#.#.#...#.#.....#.........#.....#...#...#.#.#...
.#.#####.###.###.###.###.#######.#####.###.#.#####.#.#.###.#.#.#####.#.#######.#######.#.#####.#.###.
.#.....#...#.#...#...#...#...#...#.....#...#.......#.#.#...#.#.....#...#...#...#.....#.#.........#...
.#####.###.#.#.#######.###.#.#####.#####.###########.###.###.#.###########.#.###.###.#######.#####.##
.....#.#...#.#.........#...#.......#...#.#.....#...#.......#.#.............#.....#.#.......#.#...#.#.
####.#.###.###.###.#####.#############.#.###.#.#.#.#########.#.#.#####.#.#########.#######.#.###.#.#.
.....#.....#.#.#...#.....#.....#.......#...#.#...#.#...#.....#.#.....#.#...#.#.....#.....#.#.....#.#.
.###.#######.###.###.#####.###.#.#####.###.#.#####.#.#.#.#########.#.###.###.#.###.#.#.#.#.#.#####.#.
.#.....#.#.......#...#...#.#.#.#.....#...#...#...#...#.#.........#.#...#...#.#.#.#...#.#.#.#.#.......
.#######.#.###.###.###.###.#.#.#.###.#########.#######.#########.#.###.#####.#.#.#####.###.#.#.#.####
.......#...#.#...#.........#...#.#...#...#...#.........#.......#.#...#...#.....#.....#.....#.#.#.#...
.#####.#.###.###.###########.#######.#.#.#.#.#.#############.#.#.#.#####.#.#####.###########.#.###.#.
.#.....#.....#.........#.....#.........#...#.#.#.#.....#...#.#.#.#.#...#...#...#...#.......#.#.....#.
.#.###.#####.#########.#.#####.###.#####.###.#.#.#.###.#.#.#.#.#.###.#.#.###.#####.#.#.#####.#######.
.#...#.###.......#.....#...#.....#.....#...#...#...#.#.#.#.#.#.#.....#.#.......#...#.#.....#.....#...
##.#.#.###.#######.###.###.#.#############.#####.###.#.#.#.###.#######.#######.#.###.#####.###.###.##
.#.#.#.#...#.#.....#.#...#.#.#...#.....#.........#.....#.#...........#...#.....#.#...#...#.....###...
.#.#.###.###.#.#####.#.#.#.###.#.#.###.#.#######.#####.#.###########.###.###.###.#.###.#.###.#####.##
...#.......#.#.#.....#.#.#.....#...#...#.#...#...#...#.......#.....#...#.#...#.....#...#...#...###...
.#########.#.#.###.###.#############.#####.#.#.###.#.#.#######.#####.###.#.#########.#####.#######.#.
.#.#.....#...#...#.....#.......#...#...#...#.#...#.#.#.#.....#.......#...#.#.#.......#...#.....#...#.
.#.#.###.#######.#######.###.###.#####.#.###.#####.#.#.#.###.#####.###.###.#.#.#######.#.#####.#####.
.#...#.........#.........#.#...#.......#.#.#...#...#.#.#...#.....#.#...#...#...#.#...#.#.....#.#...#.
.#####.#####.#.#####.#####.###.#.#####.#.#.#.#.#.###.#####.#####.###.#.#.###.###.#.#.#.#####.#.#.#.#.
.......#.....#.....#.........#.......#...#...#...#.........#.........#.#.....#.....#.......#.....#...
