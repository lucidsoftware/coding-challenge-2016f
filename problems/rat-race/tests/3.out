Maximum path length is 668
Maximum path length is 2100
